src/main/webapp/app/exam/manage/exams/exam-mode-picker/exam-mode-picker.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="d-flex align-items-start">
+    <div class="btn-group" [ngClass]="{ disabled: disabled }">
+        <div class="btn" [ngClass]="{ 'btn-success': !exam.testExam, 'btn-default': exam.testExam, disabled: disabled }" (click)="setExamMode(false)">
+            {{ 'artemisApp.examManagement.testExam.realExam' | artemisTranslate }}
```

Comment:
The proposal is to use `realExams` and `testExams` internally for the two exam modes. Externally `Exam` and `Test Exam` would be used (but this can be changed if desired) 
I used the camelCase here to clearly indicate the two different exam modes and to avoid confusion with other used terms. (`test exam` for example is often used (misleading in this context) to indicate a RealExam for testing.)

Reply:
The proposal is to use realExams and testExams internally for the two exam modes. Externally Exam and Test Exam would be used (but this can be changed if desired)
I used the camelCase here to clearly indicate the two different exam modes and to avoid confusion with other used terms. (test exam for example is often used (misleading in this context) to indicate a RealExam for testing.)